gui/tray.cpp

```diff
@@ -13,7 +13,7 @@ Tray::Tray(QIcon icon)
 
     connect(trayIcon, &QSystemTrayIcon::activated, this, &Tray::iconActivated);
 
-    minimizeAction = new QAction(tr("Mi&nimize"), this);
+    minimizeAction = new QAction(tr("&Minimize"), this);
```

Comment:
The `&` should stay where it is, it means that `alt + I` will minimize it, this will move the command to `alt + M`

Reply:
ok